test/unit/mdc-chips/mdc-chip-set.foundation.test.js

```diff
@@ -184,3 +185,20 @@ test('in filter chips, on custom MDCChip:interaction event deselects selected ch
   td.verify(chipA.foundation.setSelected(false));
   assert.equal(foundation.selectedChips_.length, 0);
 });
+
+test('on custom MDCChip:removal event removes chip', () => {
```

Comment:
Do we not have a test in this file for addChip? (We're testing it at the component level, but IIUC that's mocking the foundation, so we should be testing it here too.)